Orange/widgets/data/owfile.py

```diff
@@ -86,6 +85,17 @@ def __init__(self):
         if len(self.recent_files) > 0:
             self.open_file(self.recent_files[0])
 
+    def _relocate_recent_files(self):
+        rec = []
+        for fn in self.recent_files:
+            if os.path.exists(fn):
+                rec.append(fn)
+            else:
+                name = os.path.split(fn)[1]
```

Comment:
Should we also check datasets included with Orange?

https://github.com/biolab/orange3/blob/master/Orange/data/table.py#L31